### PR TITLE
0.11.0

### LINKED ISSUE #136: Table option to suppress checkbox column
I see many uses of Table to just view the results in a table format without the need to have the table drive further down-selection. Although the checkboxes on the left are great for having the table drive the next steps (selected rows) it would be great to have an option to have the table without this column.

Perhaps an option like 'selectable' with a default of true?

### PR DESCRIPTION
**Pull request by Fil**
This…

- Fixes the `+esm` entry point to use the `inputs-3a86ea` namespace instead of `__ns__`
- Publishes the Inputs stylesheet to `dist/index.css`
- … thereby making Inputs more easily used in a vanilla environment (including Framework)
- Removes `viewof` from README examples
- Links to Framework examples, too
- Adopts Vite, replacing deprecated Snowpack
- Upgrades various dependencies
- Upgrades test workflow to use Node 20
- Adds publish-to-npm workflow
- Adds the **select** option to Inputs.table

Fixes #136.
Fixes https://github.com/observablehq/framework/issues/344.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
 # Observable Inputs
 
-**Observable Inputs** are lightweight user interface components — buttons, sliders, dropdowns, tables, and the like — to help you explore data and build interactive displays in [Observable notebooks](https://observablehq.com). Each input can be used as an [Observable view](https://observablehq.com/@observablehq/introduction-to-views). For example, to allow a number *x* to be manipulated by a slider:
+**Observable Inputs** are lightweight user interface components — buttons, sliders, dropdowns, tables, and the like — to help you explore data and build interactive displays in [Observable](https://observablehq.com). Each input exposes a _value_ property that reflects its current value, and emits an _input_ event when the value changes. For example, to allow a number *x* to be manipulated by a slider:
+
+```js
+const slider = Inputs.range([0, 100]);
+slider.addEventListener((event) => {
```
Comment (mbostock) [inline README.md:7]: Missing an `"input"` here. 🤦 I’ll fix.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,15 @@
...
+  // do something with event.value
```
Comment (mbostock) [inline README.md:8]: This should be `slider.value` or `event.target.value`. 🤦 I’ll fix this too…

File: README.md [hunk 3]
```diff
@@ -12,10 +21,18 @@ Now you can reference the live value of *x* in any cell, *e.g.*:
 md`The value of *x* is ${x}.`
 ```
 
-Any cell that references *x* will run automatically when the *viewof x* slider is moved. For live examples, see:
+Any cell that references *x* will run automatically when the slider is moved. For live examples, see:
 
 https://observablehq.com/@observablehq/inputs
 
+In [Observable Framework](https://observablehq/com/framework), the reactivity is ensured with the *view* function, that both displays the input and returns the value generator. Write:
```
Comment (mbostock) [inline README.md:28]: The URL is broken here: `observablehq/com` instead of ``observablehq.com`. 🤦 I’ll fix… 😓

**Comment by mbostock** [comment]
I’m turning this into an 0.11 release with a bunch of other chore improvements…

**Review by mbostock — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (Fil) [reply README.md]: oops that was a bad day!

File: README.md [hunk 4]
```diff
@@ -1,20 +1,44 @@
...
+// Listen and respond to the user dragging the slider.
+slider.addEventListener("input", (event) => {
+  console.log(event.target.value);
+});
+
+</script>
 ```
 
-Now you can reference the live value of *x* in any cell, *e.g.*:
+Observable Inputs are designed to be especially convenient in [Observable Framework](https://observablehq/com/framework) and [Observable notebooks](https://observablehq/). In Framework, use the *view* function to display an input and expose its value as a top-level [reactive variable](https://observablehq.com/framework/reactivity):
```
Comment (Fil) [inline README.md:29]:
```suggestion
Observable Inputs are designed to be especially convenient in [Observable Framework](https://observablehq.com/framework) and [Observable notebooks](https://observablehq.com/). In Framework, use the *view* function to display an input and expose its value as a top-level [reactive variable](https://observablehq.com/framework/reactivity):
```

**Review by Fil — COMMENTED** [review]
link typo creep

**Comment by mbostock** [comment]
Thanks for finding and fixing more typos! If this looks good to you, please merge and I can work on release notes?

**Pull request merged**